dcrjson/chainsvrcmds.go

```diff
@@ -158,18 +158,18 @@ const (
 
 	// EstimateSmartFeeConservative potentially returns
 	// a conservative result.
-	EstimateSmartFeeConservative = "conservative"
+	EstimateSmartFeeConservative EstimateSmartFeeMode = "conservative"
 )
 
 // EstimateSmartFeeCmd defines the estimatesmartfee JSON-RPC command.
 type EstimateSmartFeeCmd struct {
 	Confirmations int64
-	Mode          EstimateSmartFeeMode
+	Mode          *EstimateSmartFeeMode `json:"mode,omitempty"`
```

Comment:
Commands use positional paramters, so no need to name the parameter or omitempty.  Instead, it should have the default value though akin to `GetBlockCommand`.

e.g.
```Go
Mode          *EstimateSmartFeeMode `jsonrpcdefault:"conservative"`
```

Reply:
Had to quote it to get the register function to read it correctly...

Reply:
Yep, it's a string param, so should have to be quoted.  My fault leaving that off.